code/game/objects/effects/spawners/random_spawners.dm

```diff
@@ -193,6 +193,9 @@
 	result = list(/datum/nothing = 1,
 		/obj/machinery/porta_turret/syndicate/exterior = 1)
 
+/obj/effect/spawner/random_spawners/syndicate/turret/external/always
+	name = "50pc ext turret"
+	result = list(/obj/machinery/porta_turret/syndicate/exterior)
```

Comment:
Please just map these in directly, there is no reason for a 100% chance spawner

Reply:
With ones i should map in?

Reply:
I dont want to map normal ones because they have only 50% chance to spawn

Reply:
ah, nvm i found it